src/libs/ReportUtils.ts

```diff
@@ -641,163 +818,117 @@ function isAllowedToComment(report) {
+function isPolicyAdmin(policyID: string, policies: OnyxCollection<Policy>): boolean {
+    const policyRole = policies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`]?.role;
 
     return policyRole === CONST.POLICY.ROLE.ADMIN;
 }
 
 /**
  * Returns true if report has a single participant.
- *
- * @param {Object} report
- * @returns {Boolean}
  */
-function hasSingleParticipant(report) {
-    return report && report.participantAccountIDs && report.participantAccountIDs.length === 1;
+function hasSingleParticipant(report: OnyxEntry<Report>): boolean {
+    return report?.participantAccountIDs?.length === 1;
 }
 
 /**
  * Checks whether all the transactions linked to the IOU report are of the Distance Request type
  *
- * @param {string|null} iouReportID
- * @returns {boolean}
  */
-function hasOnlyDistanceRequestTransactions(iouReportID) {
+function hasOnlyDistanceRequestTransactions(iouReportID: string | undefined): boolean {
     const allTransactions = TransactionUtils.getAllReportTransactions(iouReportID);
-    return _.all(allTransactions, (transaction) => TransactionUtils.isDistanceRequest(transaction));
+    return allTransactions.every((transaction) => TransactionUtils.isDistanceRequest(transaction));
 }
 
 /**
  * If the report is a thread and has a chat type set, it is a workspace chat.
- *
- * @param {Object} report
- * @returns {Boolean}
  */
-function isWorkspaceThread(report) {
+function isWorkspaceThread(report: OnyxEntry<Report>): boolean {
     return Boolean(isThread(report) && !isDM(report));
 }
 
-/**
- * Returns true if reportAction has a child.
- *
- * @param {Object} reportAction
- * @returns {Boolean}
- */
-function isThreadParent(reportAction) {
-    return reportAction && reportAction.childReportID && reportAction.childReportID !== 0;
-}
-
 /**
  * Returns true if reportAction is the first chat preview of a Thread
- *
- * @param {Object} reportAction
- * @param {String} reportID
- * @returns {Boolean}
  */
-function isThreadFirstChat(reportAction, reportID) {
-    return !_.isUndefined(reportAction.childReportID) && reportAction.childReportID.toString() === reportID;
+function isThreadFirstChat(reportAction: OnyxEntry<ReportAction>, reportID: string): boolean {
+    return reportAction?.childReportID?.toString() === reportID;
 }
 
 /**
  * Checks if a report is a child report.
- *
- * @param {Object} report
- * @returns {Boolean}
  */
-function isChildReport(report) {
+function isChildReport(report: OnyxEntry<Report>): boolean {
     return isThread(report) || isTaskReport(report);
 }
 
 /**
  * An Expense Request is a thread where the parent report is an Expense Report and
  * the parentReportAction is a transaction.
- *
- * @param {Object} report
- * @returns {Boolean}
  */
-function isExpenseRequest(report) {
-    if (isThread(report)) {
+function isExpenseRequest(report: OnyxEntry<Report>): boolean {
+    if (report && isThread(report)) {
```

Comment:
```suggestion
    if (isThread(report)) {
```
I think isThread will take care of empty report